### PR TITLE
Add 'serving images' section, minor corrections and links

### PR DESCRIPTION
**Pull request by joseluisbolos**

### CODE REVIEW

**Comment by nickcall** [comment]
Makes sense to me.

File: practices/images-best-practices.md [hunk 1]
```diff
@@ -38,4 +45,25 @@ Always use the maximum compression level (9).
...
+
+For images that are going to be included in a repo, you can use tools like [ImageOptim](https://imageoptim.com/mac) to optimise them and strip the metadata before committing them.
+
+There's also an [ImageOptim-CLI](https://github.com/JamieMason/ImageOptim-CLI) that can be integrated with CI systems.
+
+## Serving images
+
+We work under the assumption that images are encoded only once but decoded many times. Therefore, encoding times are usually secondary to the savings in e.g. file size.
+
+When optimising non-content images that are going to be commited to a repo, make sure you always use the maximum settings if you're using lossless optimisations.
```
Comment (dotcode) [inline practices/images-best-practices.md]: committed

File: practices/images-best-practices.md [hunk 2]
```diff
@@ -38,4 +45,25 @@ Always use the maximum compression level (9).
...
+
+For content images, we encourage using an image server like [Immagine](https://github.com/springernature/immagine) or an image service like [Kraken.io](https://kraken.io/) in order to:
+
+* Ensure that images are optimised automatically.
+* Ensure that appropriate cache headers are set.
+
+Also, future improvements to these tools will automatically be reflected in all of our images being served, instead of them having to be re-encoded.
```
Comment (dotcode) [inline practices/images-best-practices.md]: Given this has a public audience, say "the images" rather than "our images".

**Comment by joseluisbolos** [comment]
Sorted 👍

**Pull request merged**